### ISSUE TITLE
zero vector order in the domain label vector

### ISSUE THREAD

**Issue by yy97831** [yunjey/StarGAN#135]
Hi can i change the order of the zero vector in the domain label from :
c_org = torch.cat([zero, c_org, mask], dim=1)                    
c_trg = torch.cat([zero, c_trg, mask], dim=1)

to 

c_org = torch.cat([c_org,  zero, mask], dim=1)                    
c_trg = torch.cat([ c_trg, zero, mask], dim=1)

it should be the same as the network will ignore the zeros anyway, am I right?

thanks
